### PR TITLE
Working buildpack decorator that is configurable

### PR DESCRIPTION
**Pull request by keitwb**

### CODE REVIEW

File: bin/detect [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exit 1
```
Comment (mjtieman) [inline bin/detect:3]: What does this script do?

File: bin/decorate [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ROOT_DIR=$(dirname $BIN_DIR)
+BUILD_DIR=$1
+
+if [ -d $BUILD_DIR/.signalfx ]
+then
+  echo 'signalfx'
+  exit 0
+else
+  exit 1
+fi
```
Comment (mjtieman) [inline bin/decorate:14]: What is the purpose of this script?

File: bin/detect (hunk 1 above)
Comment (keitwb) [reply bin/detect:3]: Tells the meta-buildpack that this isn't a regular buildpack.

File: bin/decorate (hunk 2 above)
Comment (keitwb) [reply bin/decorate:14]: Tells meta-buildpack whether this decorator applies based on whether they have signalfx config in their app dir.

**Review by mjtieman — APPROVED** [review]

**Pull request merged**